test/unit/member_status_test.js

```diff
@@ -24,6 +24,22 @@ var _ = require('underscore');
 var test = require('tape');
 
 var Member = require('../../lib/membership/member');
+var Ringpop = require('../../index');
+
+var ALL_STATUSES = getAllStatuses();
```

Comment:
since we have `underscore` already as a dependency you might choose to use `_.values(Member.Status)` here instead of defining a function for 1 time use to do the same.